aten/src/ATen/cpu/vec/vec_base.h

```diff
@@ -66,27 +66,26 @@ Windows llvm will not have this defination.
 #endif
 #define VECTOR_WIDTH 64
 #define int_vector __m512i
-#elif defined(__aarch64__) && !defined(CPU_CAPABILITY_SVE) // CPU_CAPABILITY_AVX512
-// SVE code expects 256-vectors; leave that set for SVE?
+#elif defined(CPU_CAPABILITY_AVX2) || defined(CPU_CAPABILITY_SVE256)
 #if defined(__GNUC__)
-#define __at_align__ __attribute__((aligned(16)))
+#define __at_align__ __attribute__((aligned(32)))
 #elif defined(_WIN32)
-#define __at_align__ __declspec(align(16))
+#define __at_align__ __declspec(align(32))
 #else
 #define __at_align__
 #endif
-#define VECTOR_WIDTH 16
-#else // CPU_CAPABILITY_AVX512
+#define VECTOR_WIDTH 32
+#define int_vector __m256i
+#else // CPU_CAPABILITY_SVE128 || CPU_CAPABILITY_DEFAULT (NEON)
```

Comment:
nit: this logic is getting complicated. perhaps it would be best to make this a proper `#elif`?

Reply:
Yeah agree. I have made the changes and made it more readable and added a proper #elif to separate the macros.
Can you please check now.